src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -3213,14 +3213,16 @@ private BoundExpression BindArgumentExpression(BindingDiagnosticBag diagnostics,
         }
 
 #nullable enable
-        private void CoerceArguments<TMember>(
+        private void CheckAndCoerceArguments<TMember>(
             MemberResolutionResult<TMember> methodResult,
-            ArrayBuilder<BoundExpression> arguments,
+            AnalyzedArguments analyzedArguments,
             BindingDiagnosticBag diagnostics,
-            BoundExpression? receiver)
+            BoundExpression? receiver,
+            bool interpolatedStringHandler = false)
```

Comment:
>interpolatedStringHandler [](http://example.com/codeflow?start=17&length=25)

Consider using the name that reflects the purpose of the parameter. For example, "suppressRefReadonlyFeatureWarnings".